packages/app/components/onboarding/hcaptcha/index.web.ts

```diff
@@ -0,0 +1 @@
+export { Challenge } from "./challenge.web";
```

Comment:
we can delete this file since anyway the `hcaptcha/index.ts` file will import the `challenge.web` file on web.